runtime/common/addresslocation.go

```diff
@@ -80,10 +80,11 @@ func (l AddressLocation) MeteredID(memoryGauge MemoryGauge) LocationID {
 }
 
 func (l AddressLocation) TypeID(memoryGauge MemoryGauge, qualifiedIdentifier string) TypeID {
-	return NewMeteredTypeID(
+	return idLocationTypeID(
```

Comment:
Can the old method be replaced with the new one?

Is the new method, and the old one essentially produce the same output? If the old method will not be removed, then maybe add a test case to ensure both produce identical results

Reply:
It couldn't be removed yet, as REPL, string, and identifier locations were still relying on it. 
I optimized those as well in cbbad360e4d20444fe67403f5bad41f25a4bfcd5, and also removed the unnecessary location ID code along the way.